### PR TITLE
Backfill missing Description metadata, make it required

### PR DESCRIPTION
**Pull request by christian-dinh**
### Description

`Description` frontmatter attributes are now required! Missing descriptions have been backfilled via script using the same method we were using to generate them on the site itself. Updated docs to reflect changes.

### Type of Change

- [ ] Adding a new entry
- [x] Editing an existing entry (fixing a typo, bug, issues, etc)
- [x] Updating the documentation

### Checklist

- [x] My entry follows the Codecademy Docs style guide
- [x] I have performed a self-review of my own writing and code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have checked my entry and corrected any misspellings
- [ ] All writings are my own

### Codecademy Username

If you want your Codecademy username be displayed in your entry (feature coming in October), make sure to have your Codecademy user profile linked to your GitHub:

1. Go to your Codecademy dashboard.
2. Click on your profile image in the top-right and then choose "Profile".
3. Then click "Edit Profile".
4. In the GitHub field, add your GitHub profile URL.

### CODE REVIEW

**Comment by christian-dinh** [comment]
note: definitely holding off on this until we know for sure docs is back up and stable!

File: documentation/content-standards.md [hunk 1]
```diff
@@ -51,39 +51,39 @@ To give an example, an entry titled "What is Three.js?" should have the file nam
...
-- JavaScript: ```js
-- PHP: ```php
-- Pseudo: ```pseudo
-- Python: ```py
-- R: ```r
-- Ruby: ```rb
-- Sass: ```scss
-- Shell/Program Output: ```shell
-- SQL: ```sql
+All code blocks should be delineated by triple backticks (\`\`\`)
```
Comment (MattSchiller) [inline documentation/content-standards.md:60]: I presume this in intentional?

File: content/markdown/concepts/emphasis/emphasis.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 Title: 'Emphasis'
+Description: 'Emphasis can be added by making text bold or italic. To bold text, add two asterisks  or underscores __ before and after a word or phrase. To bold the middle of a word for emphasis, add two asterisks without spaces around the letters. markdown bold text*  markdown bold text*'
```
Comment (MattSchiller) [inline content/markdown/concepts/emphasis/emphasis.md:3]: This is actually an artifact of a bug lol

Can just remove everything after `letters.`

**Review by MattSchiller — APPROVED** [review]
OMG THANK YOU

Left some comments where the code looked a bit strange to me.

Also, just checking, we're still planning to go over with humans to update these at _some_ point, right? There are likely more minor parsing issues.

File: documentation/content-standards.md (hunk 1 above)
Comment (christian-dinh) [reply documentation/content-standards.md:60]: gonna be real I'm very surprised that this file [renders reasonably](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md) as currently written, but I didn't check before making these changes. I guess they're not necessary 🤷

**Comment by christian-dinh** [comment]
@MattSchiller yeah I think that would be ideal, on top of parsing issues I imagine the content of a lot of these is pretty suboptimal as well

**Review by dougyd92 — APPROVED** [review]
Cool!
I was going to ask where the descriptions show up, but it's right there in the documentation: "used in search engine results and content previews". Thanks!

**Pull request merged**